Makefile

```diff
@@ -19,24 +19,24 @@ docs/index.html:	docs/*.adoc docs/docinfo*.html ${STYLESDIR}/${STYLESHEET}
 			docs/index.adoc
 
 test-clj:
-			clojure -Atest -e deprecated
```

Comment:
it looks like it was skipping tests marked as deprecated, should we still do the same?
I don't think they are actually breaking unless I turn on the orchestra plugin which does all the spec checking, in which case they do break.